src/pages/graphql/payment-services-extension/queries/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Payment Services extension queries
+description: Learn how to use the GraphQL queries to integrate the Payment Services extension.
+keywords:
+  - GraphQL
+  - Payments
+---
+
+# Payment Services extension queries
+
+The Payment Services extension queries enable you to retrieve payment-related data, such as transaction details, payment statuses, and more, allowing you to efficiently manage and monitor your payment services extension.
```

Comment:
```suggestion
The queries described in this section allow you to retrieve payment-related data, such as transaction details and payment statuses so that you can manage and monitor the operations of the Payment Services extension.
```